### PR TITLE
Added the ability to filter by documents' decision status

### PR DESCRIPTION
**Pull request by Omri-Levy**
### Description
Elaborate on the subject, motivation, and context.

### Related issues
 * Provide a link to each related issue.

### Breaking changes
 * Describe the breaking changes that this pull request introduces.

### How these changes were tested
 * Describe the tests that you ran to verify your changes, including devices, operating systems, browsers and versions.

### Examples and references
 * Links, screenshots, and other resources related to this change.

### Checklist
- [] I have read the [contribution guidelines](CONTRIBUTING.md) of this project
- [] I have read the [style guidelines](STYLE_GUIDE.md) of this project
- [] I have performed a self-review of my own code
- [] I have commented my code, particularly in hard-to-understand areas
- [] I have made corresponding changes to the documentation
- [] My changes generate no new warnings and errors
- [] New and existing tests pass locally with my changes

### CODE REVIEW

File: services/workflows-service/prisma/schema.prisma [hunk 1]
```diff
@@ -154,6 +154,8 @@ enum WorkflowRuntimeDataStatus {
 
 model WorkflowRuntimeData {
   id                        String                    @id @default(cuid())
+  // Potentially use an enum
+  tasksStatus               String                    @default("pending")
```
Comment (alonp99) [inline services/workflows-service/prisma/schema.prisma]: Make nullable, remove default

File: services/workflows-service/src/workflow/utils/pick-tasks-status.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { safeEvery, someDocumentDecisionStatus, TDocument } from '@ballerine/common';
+import { DocumentDecisionStatus } from '@/workflow/types';
+
+export const pickTasksStatus = (documents: Array<TDocument>) => {
```
Comment (alonp99) [inline services/workflows-service/src/workflow/utils/pick-tasks-status.ts]: Name change (calculate compute task status from documents)

File: services/workflows-service/src/workflow/utils/toPrismaWhere.ts [hunk 3]
```diff
@@ -1,8 +1,10 @@
 import { Prisma, WorkflowRuntimeDataStatus } from '@prisma/client';
+import { TDocumentDecisionStatus } from '@/workflow/types';
 
 type Filters = {
   assigneeId?: (string | null)[];
   status?: WorkflowRuntimeDataStatus[];
+  tasksStatus?: Array<TDocumentDecisionStatus>;
 };
 
 export const toPrismaWhere = (filters: Filters): Prisma.WorkflowRuntimeDataWhereInput => {
```
Comment (alonp99) [inline services/workflows-service/src/workflow/utils/toPrismaWhere.ts:10]: @MatanYadaev Should it be in the repository?

**Review by alonp99 — APPROVED** [review]

File: services/workflows-service/src/workflow/utils/toPrismaWhere.ts (hunk 3 above)
Comment (MatanYadaev) [reply services/workflows-service/src/workflow/utils/toPrismaWhere.ts:10]: Yes.

**Pull request closed without merging**